lib/tasks/demo.js

```diff
@@ -258,8 +258,10 @@ module.exports = function(gulp, config) {
 		}
 
 		demosConfig.demos.forEach(function(demoConfig) {
-			// Extend an empty object to avoid changing the value of defaultDemoConfig
-			demos.push(extend(true, {}, defaultDemoConfig, demosConfig.demosDefaults || demosConfig.options, demoConfig));
+			if (!config.buildDemo || (config.buildDemo && config.buildDemo === demoConfig.name)) {
```

Comment:
I didn't add it to the readme because I thought it was only a build service thing and better to be kept private